packages/api/endpoints/executions.js

```diff
@@ -359,6 +356,44 @@ async function bulkDeleteExecutionsByCollection(req, res) {
   return res.status(202).send({ id: asyncOperationId });
 }
 
+const bulkArchiveExecutionsSchema = z.object({
+  updateLimit: z.number().min(1).optional().default(10000),
+  batchSize: z.number().min(1).optional().default(1000),
+  expirationDays: z.number().optional().default(365),
+});
+const parseBulkArchiveExecutionsPayload = zodParser('bulkChangeCollection payload', bulkArchiveExecutionsSchema);
+/**
+ * Start an AsyncOperation that will archive a set of executions in ecs
+ */
+async function bulkArchiveExecutions(req, res) {
+  const payload = parseBulkArchiveExecutionsPayload(req.body);
+  if (isError(payload)) {
+    return returnCustomValidationErrors(res, payload);
+  }
+  const asyncOperationId = uuidv4();
+  const asyncOperationEvent = {
+    asyncOperationId,
+    callerLambdaName: getFunctionNameFromRequestContext(req),
+    lambdaName: process.env.ArchiveRecordsLambda,
+    description: 'Launches an ecs task to archive a batch of executions',
+    operationType: 'Bulk Execution Archive',
+    payload: {
+      config: {
+        ...payload,
+        recordType: 'executions',
```

Comment:
Is it possible to use a single type for recordType, such as 'execution' or 'granule'? We already have simliar type defined in the baseline:
export type RecordType = 'execution' | 'granule' | 'pdr';
https://github.com/nasa/cumulus/blob/master/packages/types/message.d.ts#L4